library/Fission/Web/Swagger/Docs.hs

```diff
@@ -53,3 +54,8 @@ ping :: Swagger -> Swagger
 ping = applyTagsFor ops  ["Ping" & description ?~ "Check for liveness"]
   where
     ops = subOperations (Proxy :: Proxy Web.PingRoute) (Proxy :: Proxy Web.API)
+
+register :: Swagger -> Swagger
+register = applyTagsFor ops  ["Register" & description ?~ "Register and provision a user"]
```

Comment:
Just took a look at this in Swagger, and it makes me think that we should rename this whole thing to `User` routes. Other than it's "standard" REST, an example of why: let's say we later allow users to add an email address. If they want to update that, we'll want them to make a `PUT` request to update the same resource. I don't think that it's conceptually a "registration" that they're updating; it's the user.

This would be a minor change to this PR: change `/register` to `/user`, update the module names, and all good.

Thoughts?

Reply:
Also, heads up: I'm going to scrap the `docs` vs `all_docs` distinction. Sure, consumers don't need to know what the Heroku endpoint looks like, but it really helps when you're running one of these yourself. I mention it here because the `/register` endpoint got added to `all_docs` but not `docs`. It's too easy to forget, and unnecessarily complicates the routes if we're not using it. I was going to DRY up some of that code, but this seems more sane.

Reply:
yeah fan of this^ I was thinking after the fact of changing everything to `/auth` (ie `/auth/register`, `auth/verify`, etc) but I like `/user` better